nestkernel/connection_manager.cpp

```diff
@@ -1488,11 +1488,23 @@ nest::ConnectionManager::reserve_connections( const thread tid,
   const synindex syn_id,
   const size_t count )
 {
+  /*
```

Comment:
do i understand correctly that we want to keep the code above that computes the expected number of connections, and also this todo in the release?